docs/roadmap/README.md

```diff
@@ -7,90 +7,23 @@ parent:
 
 # Previous Releases
 
-Please see the table below for library versions and other dependencies.  
```

Comment:
Deleted as this content is in the release notes. Don't want to maintain copies of release notes in other docs.